src/widget/translator.cpp

```diff
@@ -72,6 +73,11 @@ void Translator::translate()
         QCoreApplication::installTranslator(translator);
     }
 
+    // Added to fix #3542
```

Comment:
rather than mentioning issue number, it's better to describe what it actually fixes